File: Java/loops6.java
// Multiplication table of a number
import java.util.Scanner;

public class loops6 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter N: ");
        int n = sc.nextInt();

        for (int i = 1; i <= 10; i++) {
            int result = n * i;
            System.out.println(n + " * " + i + " = " + result);
        }
    }
}

File: Java_Practice/CPP/loops6.cpp
// Multiplication table of a number
#include <iostream>
using namespace std;

int main() {
    int n, i, result;
    cout << "ENTER N : ";
    cin >> n;

    for (i = 1; i <= 10; i++) {
        result = n * i;
        cout << n << " * " << i << " = " << result << endl;
    }

    return 0;
}
